engine/build.gradle

```
@@ -33,52 +33,58 @@ buildscript {
 }
 
 dependencies {
-    compile "com.badlogicgames.gdx:gdx:$gdxVersion"
-    compile "com.badlogicgames.gdx:gdx-box2d:$gdxVersion"
-    compile "com.badlogicgames.gdx:gdx-controllers:$gdxVersion"
+    api group: 'org.slf4j', name: 'slf4j-log4j12', version: '1.7.25'
```

The engine should not depend on a specific logging back-end of slf4j. Log4j caused errors at runtime with the android facade when I tried to run this code. Only the desktop facade uses log4j for logging so far. The engine should depend on `slf4j-api` instead. See http://www.slf4j.org/manual.html#swapping for more information.
```suggestion
    api group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
```